modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -3091,6 +3133,53 @@ private void _testAddObjectEntryAccountEntryRestriction(
 			ObjectActionKeys.ADD_OBJECT_ENTRY, _buyerRole);
 	}
 
+	private void _testGetObjectEntriesWithAccountEntryRestrictedAndSearch()
+		throws Exception {
+
+		_addResourcePermission(ObjectActionKeys.ADD_OBJECT_ENTRY, _buyerRole);
+
+		AccountEntry accountEntry1 = _addAccountEntry();
+
+		PermissionThreadLocal.setPermissionChecker(
```

Comment:
Maybe you have to store the ThreadLocal original values before set the new ones and then set them back in a finally statement. Can you check how other test classes does that?

Reply:
@guilhermedcamacho  Right after this test, there is a moment that the test set another new user in threadLocal, so I believe have no issue about it here